### DISCUSSION TITLE
我要选出chatgpt能用的有办法吗

### DISCUSSION THREAD

**Discussion by lozn00** [XIU2/CloudflareSpeedTest#514] [提问]
### 功能需求

我要选出chatgpt能用的有办法吗

### 预期目标

我要选出chatgpt能用的有办法吗

**Comment by charSLee013** [comment]
提供另外一个思路，使用[warp-cli](https://developers.cloudflare.com/warp-client/get-started/linux/)作为代理去访问OpenAI，这样可以绕过CF盾的同时免去优选IP的烦恼

**Comment by Davidasx** [comment]
之前用cloudflare pages自建代理，是可以访问ChatGPT的，但是要刷新三次。
